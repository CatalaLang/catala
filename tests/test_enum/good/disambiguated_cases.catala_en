## Article

```catala
declaration enumeration E:
  -- Case1

declaration enumeration F:
  -- Case1 content integer
  -- Case2

declaration scope A:
  context e content E
  context f content F
  context x content integer

scope A:
  definition e equals E.Case1
  definition f equals F.Case1 content 2
  definition x equals match f with pattern
    -- F.Case1 of i : i
    -- Case2 : 3
```
