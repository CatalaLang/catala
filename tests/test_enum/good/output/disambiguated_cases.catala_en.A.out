[RESULT] Computation successful! Results:
[RESULT] e = Case1 ()
[RESULT] f = Case1 2
[RESULT] x = 2
